app/models/spree/app_configuration_decorator.rb

```diff
@@ -0,0 +1,5 @@
+Spree::AppConfiguration.class_eval do
```

Comment:
I prefer avoiding `class_eval`. Instead I'd create a new configuration class. This is my point of reference:
http://guides.spreecommerce.org/developer/preferences.html#general-settings

https://github.com/spree-contrib/spree_address_book/blob/master/lib/spree/address_book_configuration.rb

https://github.com/spree-contrib/spree_address_book/blob/master/lib/spree_address_book.rb#L8